src/DataStructures/DataBox/TagTraits.hpp

```diff
@@ -18,45 +18,52 @@ struct ComputeTag;
 /// \endcond
 
 namespace db {
-// @{
 /*!
  * \ingroup DataBoxGroup
- * \brief Check if `Tag` derives off of db::ComputeTag
+ * \brief Check if `Tag` derives off of db::ComputeTag.
+ * \see is_compute_tag_v ComputeTag
  */
 template <typename Tag, typename = std::nullptr_t>
 struct is_compute_tag : std::false_type {};
-/// \cond HIDDEN_SYMBOLS
+/// \cond
 template <typename Tag>
 struct is_compute_tag<Tag, Requires<std::is_base_of_v<db::ComputeTag, Tag>>>
     : std::true_type {};
 /// \endcond
```

Comment:
Not needed for this PR, but this would be simpler as
```c++
template <typename Tag>
struct is_compute_tag : std::bool_constant<std::is_base_of_v<db::ComputeTag, Tag>> {};
```